src/test/java/seedu/address/logic/parser/ParserUtilTest.java

```diff
@@ -125,6 +219,22 @@ public void parseEmail_validValueWithWhitespace_returnsTrimmedEmail() throws Exc
         assertEquals(expectedEmail, ParserUtil.parseEmail(emailWithWhitespace));
     }
 
+    @Test
+    public void parseDescription_null_throwsNullPointerException() {
```

Comment:
InvalidDescription would be "", cuz we kinda catch every word, no need to change tho unless we want to apply more constraints to it.

Reply:
Thanks. I have added the test for invalid description in this commit 9b512f3